project/cmake/KodiConfig.cmake.in

```diff
@@ -12,6 +12,9 @@ endif()
 if(NOT @APP_NAME_UC@_LIB_DIR)
   set(@APP_NAME_UC@_LIB_DIR @APP_LIB_DIR@)
 endif()
+if(NOT @APP_NAME_UC@_DATA_DIR)
+  set(@APP_NAME_UC@_DATA_DIR @APP_DATA_DIR@)
```

Comment:
`@APP_DATA_DIR@` is not being set when `project/cmake/KodiConfig.cmake` is created - is there something I should be setting? The default `/usr/share` vallue should be used, but isn't, so `@APP_DATA_DIR@` is being written to the generated `project/cmake/KodiConfig.cmake`: http://sprunge.us/ENaK

Any ideas?